php/php.editor/src/org/netbeans/modules/php/editor/indent/FmtOptions.java

```diff
@@ -569,8 +574,9 @@ public void run() {
             } else {
                 LOGGER.warning(String.format("Can't format %s; it's not BaseDocument.", doc)); //NOI18N
             }
+            pane.setCaretPosition(caretPosition);
             pane.setIgnoreRepaint(false);
-            pane.scrollRectToVisible(visibleRectangle);
+            SwingUtilities.invokeLater(() -> pane.scrollRectToVisible(visibleRectangle));
```

Comment:
Because the preview pane is scrolled to the caret position when we change options after we scroll it anywhere.